docs/intro/tutorial02.txt

```diff
@@ -520,6 +520,8 @@ Save these changes and start a new Python interactive shell by running
     >>> q = Question.objects.get(pk=1)
 
     # Display any choices from the related object set -- none so far.
+    # As part of Django's support for model relations, q.choice_set gets
+    # created as part of Question being declared as a foreign key for Qhoice.
```

Comment:
"Qhoice" look like a typo. Did you mean "Choice"?

Reply:
Whoops, bad copy paste- fixed.